internal/pkg/errors/error_message.go

```diff
@@ -301,6 +301,7 @@ const (
 	ContextStateMismatchErrorMsg       = `context state mismatch for context "%s"`
 	ContextStateNotMappedErrorMsg      = `context state mapping error for context "%s"`
 	DeleteUserAuthErrorMsg             = "unable to delete user auth"
+	NonCcloudPlatformNameErrorMsg      = "must be Confluent Cloud platform"
```

Comment:
first look at this msg might be a bit confusing. `something` must be cloud platform?

Reply:
I see, how about 
```
trying to retrieve Confluent Cloud client for non-cloud platform name
```
?

Reply:
```suggestion
```